edit/edit.css

```diff
@@ -440,6 +471,10 @@ body[data-match-highlight="selection"] .CodeMirror-selection-highlight-scrollbar
 }
 #lint td[role="message"] {
   text-align: left;
+  max-width: 140px;
```

Comment:
I think we should keep the old code that cuts the message by max length, and additionally set the full text in cell's `title`.